### PR TITLE
Commit Summary Expansion: CSS tweaks

### PR DESCRIPTION
**Pull request by tidy-dev**
Based on #17556 

## Description
This PR is to make the vertical spacing between all the commit summary elements consistent and go through the css file removing unused rules. It also removes the commit description from multi commit selection where showing the description from the first commit doesn't make sense.

Scenarios tested:
1. Commit no description
2. Commit 1 line description
3. Commit Overflowing description
4. Commit all the metadata
5. Multiple commits selected
6. Commit not reachable.

## Release notes
Notes: no-notes

### CODE REVIEW

File: app/styles/ui/history/_expandable-commit-summary.scss [hunk 1]
```diff
@@ -198,18 +185,28 @@
...
+      margin-right: var(--spacing-half);
     }
   }
 
   .selectable {
     user-select: text;
     cursor: text;
 
-    description {
+    * {
```
Comment (tidy-dev) [inline app/styles/ui/history/_expandable-commit-summary.scss:217]: Fixing accidental replacement in last pr.

**Review by sergiou87 — DISMISSED** [review]
✨

**Comment by sergiou87** [comment]
Oops! I just saw there are conflicts to resolve. Let me know and I will take another look after they're resolved!

**Review by sergiou87 — APPROVED** [review]
Looks good! Thanks for coping with my constant changes requests (from Slack)! 😂 💖 

:shipit: 🚀

**Comment by tidy-dev** [comment]
> Looks good! Thanks for coping with my constant changes requests (from Slack)! 😂 💖

Thanks for super detailed reviews! Much better outcome. :)

**Pull request merged**